modules/400-descheduler/hooks/internal/api/v1alpha1/descheduler.go

```diff
@@ -0,0 +1,286 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Descheduler is a description of a single descheduler instance
+type Descheduler struct {
```

Comment:
How to keep these structures in sync with the openapi spec? This is an open question. In modules like log-shipper, we fix the code in both openapi and go and it is possible to make a mistake here.

Reply:
Not relevant. No code-generation anymore.